deepchecks/nlp/text_data.py

```diff
@@ -39,8 +39,11 @@ class TextData:
 
     Parameters
     ----------
-    raw_text : t.Sequence[str]
+    raw_text : t.Sequence[str], default: None
         The raw text data, a sequence of strings representing the raw text of each sample.
+    tokenized_text : t.Sequence[t.Sequence[str]], default: None
+        The tokenized text data, a sequence of sequences of strings representing the tokenized text of each sample.
+        Only relevant for task_type 'token_classification'.
```

Comment:
Maybe say that raw text will be generated by ' '.join-ing this?

Reply:
Done